Compile errors

I get some compile errors unfortunately:
```
Errors     << moveit_benchmark_suite:make /home/sis/Mikado/logs/moveit_benchmark_suite/build.make.004.log                                                                                                                                                                    
In file included from /home/sis/Mikado/src/moveit_benchmark_suite/moveit/include/moveit_benchmark_suite/benchmark_callback_loader.h:45,
                 from /home/sis/Mikado/src/moveit_benchmark_suite/moveit/src/benchmark_callback_loader.cpp:1:
/home/sis/Mikado/src/moveit_benchmark_suite/output/include/moveit_benchmark_suite/output/rviz_visualization.h:59:10: fatal error: rviz/SendFilePath.h: Datei oder Verzeichnis nicht gefunden
   59 | #include <rviz/SendFilePath.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/moveit_benchmark_suite.dir/build.make:63: CMakeFiles/moveit_benchmark_suite.dir/src/benchmark_callback_loader.cpp.o] Fehler 1
make[2]: *** Auf noch nicht beendete Prozesse wird gewartet …
In file included from /home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/yaml.h:17,
                 from /home/sis/Mikado/src/moveit_serialization/serialization/include/moveit_serialization/yaml-cpp/yaml.h:41,
                 from /home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/handler.h:43,
                 from /home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/robot.h:55,
                 from /home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/planning.h:47,
                 from /home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/resource_builder.h:46,
                 from /home/sis/Mikado/src/moveit_benchmark_suite/moveit/src/query_builders/collision_check_builder.cpp:41:
/home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/node/impl.h: In instantiation of ‘T YAML::as_if<T, void>::operator()() const [with T = collision_detection::CollisionRequest]’:
/home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/node/impl.h:156:31:   required from ‘T YAML::Node::as() const [with T = collision_detection::CollisionRequest]’
/home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/resource_builder.h:145:7:   required from ‘std::map<std::__cxx11::basic_string<char>, T> moveit_benchmark_suite::YAMLDeserializerBuilder<T>::generateResources() const [with T = collision_detection::CollisionRequest]’
/home/sis/Mikado/src/moveit_benchmark_suite/moveit/src/query_builders/collision_check_builder.cpp:86:45:   required from here
/home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/node/impl.h:131:27: error: incomplete type ‘YAML::convert<collision_detection::CollisionRequest>’ used in nested name specifier
  131 |     if (convert<T>::decode(node, t))
      |         ~~~~~~~~~~~~~~~~~~^~~~~~~~~
/home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/node/impl.h: In instantiation of ‘void YAML::Node::Assign(const T&) [with T = collision_detection::CollisionRequest]’:
/home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/node/impl.h:205:3:   required from ‘YAML::Node& YAML::Node::operator=(const T&) [with T = collision_detection::CollisionRequest]’
/home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/resource_builder.h:104:14:   required from ‘bool moveit_benchmark_suite::ResourceBuilder::validateYamlNode(const YAML::Node&) [with T = collision_detection::CollisionRequest]’
/home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/resource_builder.h:134:50:   required from ‘bool moveit_benchmark_suite::YAMLDeserializerBuilder<T>::validateResource(const YAML::Node&) [with T = collision_detection::CollisionRequest]’
/home/sis/Mikado/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/resource_builder.h:131:16:   required from here
/home/sis/Mikado/src/moveit_serialization/yaml-cpp/include/yaml-cpp/node/impl.h:227:32: error: incomplete type ‘YAML::convert<collision_detection::CollisionRequest>’ used in nested name specifier
  227 |   AssignData(convert<T>::encode(rhs));
      |              ~~~~~~~~~~~~~~~~~~^~~~~
make[2]: *** [CMakeFiles/moveit_benchmark_suite.dir/build.make:102: CMakeFiles/moveit_benchmark_suite.dir/src/query_builders/collision_check_builder.cpp.o] Fehler 1
make[1]: *** [CMakeFiles/Makefile2:262: CMakeFiles/moveit_benchmark_suite.dir/all] Fehler 2
make: *** [Makefile:141: all] Fehler 2

```
this is on Noetic with all dependencies from current git master/noetic-devel

I made two little changes which fixed one further error:
```
diff --git a/core/src/io.cpp b/core/src/io.cpp
index 5ebe17a..9105dbc 100644
--- a/core/src/io.cpp
+++ b/core/src/io.cpp
@@ -15,6 +15,7 @@
 #include <boost/uuid/uuid_io.hpp>          // for UUID generation
 #include <ros/package.h>                   // for package resolving
 #include <ros/console.h>
+#include <tinyxml2.h>
 
 #include <moveit_benchmark_suite/io.h>
 #include <moveit_benchmark_suite/handler.h>
```

and this one in an attempt to fix the include but it didn't help:
```
diff --git a/output/CMakeLists.txt b/output/CMakeLists.txt
index 9593bed..b320e8e 100644
--- a/output/CMakeLists.txt
+++ b/output/CMakeLists.txt
@@ -6,7 +6,7 @@ find_package(Eigen3 REQUIRED)
 find_package(fcl REQUIRED)
 
 find_package(catkin REQUIRED COMPONENTS
-  moveit_benchmark_suite_core
+  moveit_benchmark_suite_core rviz
 )
 
 catkin_package(
diff --git a/output/package.xml b/output/package.xml
index 65424f5..8dded85 100644
--- a/output/package.xml
+++ b/output/package.xml
@@ -9,6 +9,7 @@
        <buildtool_depend>catkin</buildtool_depend>
 
        <depend>moveit_benchmark_suite_core</depend>
+    <depend>rviz</depend>
 
        <export>
                <moveit_benchmark_suite benchmarks="${prefix}/benchmarks"/>
```

--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -6,6 +6,7 @@ find_package(Eigen3 REQUIRED)
 find_package(fcl REQUIRED)
 
 find_package(catkin REQUIRED COMPONENTS
+  cmake_modules
   eigen_conversions
   moveit_core
   moveit_ros_planning
@@ -15,6 +16,8 @@ find_package(catkin REQUIRED COMPONENTS
   tf2_ros
   urdf_to_scene
 )
+
+find_package(TinyXML2 REQUIRED)
 
 catkin_package(
   LIBRARIES
@@ -34,7 +37,10 @@ catkin_package(
 
 set(CMAKE_CXX_STANDARD 14)
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+include_directories(include
+  ${catkin_INCLUDE_DIRS}
+  ${TinyXML2_INCLUDE_DIRS}
+)
 
 add_library(${PROJECT_NAME}
   src/resource_builder.cpp

--- a/core/package.xml
+++ b/core/package.xml
@@ -8,6 +8,8 @@
 
 	<buildtool_depend>catkin</buildtool_depend>
 
+	<build_depend>cmake_modules</build_depend>
+
 	<depend>boost</depend>
 	<depend>eigen_conversions</depend>
 	<depend>moveit_core</depend>
@@ -15,6 +17,7 @@
 	<depend>moveit_ros_planning_interface</depend>
 	<depend>moveit_serialization</depend>
 	<depend>roscpp</depend>
+    <depend>tinyxml2</depend>
 	<depend>tf2_ros</depend>
 	<depend>urdf_to_scene</depend>
 

--- a/core/src/io.cpp
+++ b/core/src/io.cpp
@@ -15,6 +15,7 @@
 #include <boost/uuid/uuid_io.hpp>          // for UUID generation
 #include <ros/package.h>                   // for package resolving
 #include <ros/console.h>
+#include <tinyxml2.h>
 
 #include <moveit_benchmark_suite/io.h>
 #include <moveit_benchmark_suite/handler.h>

--- a/output/CMakeLists.txt
+++ b/output/CMakeLists.txt
@@ -7,6 +7,7 @@ find_package(fcl REQUIRED)
 
 find_package(catkin REQUIRED COMPONENTS
   moveit_benchmark_suite_core
+  rviz # depends only on 'SendFilePath.srv' service msg
 )
 
 catkin_package(

--- a/output/package.xml
+++ b/output/package.xml
@@ -8,6 +8,8 @@
 
 	<buildtool_depend>catkin</buildtool_depend>
 
+	<build_depend>rviz</build_depend>
+
 	<depend>moveit_benchmark_suite_core</depend>
 
 	<export>

